From a1fc1372df9f08e6f49d0c6aaf67b1b8e35ef959 Mon Sep 17 00:00:00 2001
From: Alaric Senat <dev.asenat@posteo.net>
Date: Mon, 22 Sep 2025 11:55:24 +0200
Subject: [PATCH] meson: libvlc: declare dependency

---
 lib/meson.build | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/lib/meson.build b/lib/meson.build
index e7b9de3..40f6073 100644
--- a/lib/meson.build
+++ b/lib/meson.build
@@ -31,3 +31,9 @@ libvlc = library('vlc',
     dependencies: [libvlccore_dep, m_lib, threads_dep],
     install: true
 )
+
+libvlc_dep = declare_dependency(
+    link_with: libvlc,
+    include_directories: vlc_include_dirs,
+    dependencies: [libvlccore_dep, m_lib, threads_dep]
+)
-- 
2.48.1

